ganga/GangaCore/test/gangaDB/db_controllers/TestContainerHandler.py

```diff
@@ -88,14 +92,47 @@ def setUp(self):
     #         )
     #         assert flag is None
 
+    # def test_a2_apptainer_backend_lifetime(self):
```

Comment:
Is this supposed to be all commented?

Reply:
The methods `test_a1_download_sif_file` and `test_a2_singularity_backend_lifetime` were already commented with the explanation "_Commented due the the error mentioned in the class doc_", so I didn't touch them.

I commented the `import os` statement in the beginning because intended to be used in these two commented methods. It was introducing lint errors. Added the same explanatory note here that the previous developer added for the two methods. Didn't want to remove it in case these methods are made live later.

I added a new method `test_a2_apptainer_backend_lifetime` as part of the actual update and commented it as well because it is pretty much an exact replica of `test_a2_singularity_backend_lifetime`  except for the replacement of singularity with apptainer. Didn't want to introduce any unintended side effect by keeping the new code open.